src/components/composite/Header.stories.js

```diff
@@ -29,11 +28,35 @@ const props = {
   },
 };
 
-storiesOf('Composite|Header', module)
-  .addParameters({ component: Header })
-  .add('default', () => <Header {...props} />)
-  .add('inverted', () => (
-    <BlueBackground>
-      <Header isInverted {...props} />
-    </BlueBackground>
-  ));
+export default {
+  component: Header,
+  title: 'Composite/Header',
+};
+
+export const Basic = args => <Header {...args} />;
+Basic.storyName = 'default';
+Basic.args = {
+  firstChapter: 'get-started',
+  githubUrl: 'https://github.com/chromaui/learnstorybook.com',
+  guides: {
+    edges: [
+      {
+        node: {
+          frontmatter: {
+            title: 'Guide title',
+            description: 'Guide description',
+          },
+          fields: {
+            slug: 'guide-slug',
+          },
+        },
+      },
+    ],
+  },
+};
+export const Inverted = () => (
```

Comment:
convert to Args format and spread in `Basic.args`?

Reply:
This was fixed in a prior commit. Thanks for the time reviewing it.